public/data/Chaitanya7666.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "Chaitanya",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/Chaitanya7666.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Chaitanya7666",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/chaitanya7666",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "December 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing"
```

I suggest a little touch to make it a bit more like a sentence than a heading 😉 

```suggestion
      "description": "Started freelancing!"
```